network/benchmarks/netperf/launch.go

```diff
@@ -73,6 +74,8 @@ func init() {
 	flag.StringVar(&netperfImage, "image", "sirot/netperf-latest", "Docker image used to run the network tests")
 	flag.StringVar(&kubeConfig, "kubeConfig", "",
 		"Location of the kube configuration file ($HOME/.kube/config")
+	flag.BoolVar(&cleanupOnly, "cleanup", false,
```

Comment:
Let change the description to be more explicit: "Run the cleanup resources phase only (use this flag to clean up orphaned resources from a test run)"